### ISSUE TITLE
SyntaxError: JSON.parse: unexpected character Error with `--base-url` option

### ISSUE THREAD

**Issue by snilaxan** [rickbergfalk/sqlpad#318]
Hi All, 

Started sqlpad app using following command with `--base-url` :
`sqlpad --dir /ebs/sqlpad/data --port 3000 --ip 0.0.0.0 --base-url '/sqlpad' --save`

But when we enter the correct username and password, Getting following Syntax error message on UI.
`SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data`

Please find the screenshot of the error message :
![image](https://user-images.githubusercontent.com/3293837/37197409-b39513b4-23a0-11e8-96ae-0ae10f1edd7f.png)

Please let us know, How properly use the `--base-url` option to fix this issue.

**Comment by rickbergfalk** [comment]
Sorry for the delay in getting to this @snilaxan - it looks like you are setting it correctly. I'll see if I can reproduce and figure out what is going on

**Comment by snilaxan** [comment]
Thanks, @rickbergfalk for your reply. Please find additional details about the environment.

```
root :~$ nodejs -v
v8.10.0
```
```
root :~$ npm -v
5.7.1
```
```
root :~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.4 LTS
Release:	16.04
Codename:	xenial
```

**Comment by rickbergfalk** [comment]
This should be fixed in version 2.5.2 now published to npm and docker. Let me know if you're still running into this after the update

**Comment by snilaxan** [comment]
Thanks a lot for fixing this issue.
